### PR TITLE
[Doctests] Add `configuration_vit_mae.py` and `configuration_yoso.py`

### PR DESCRIPTION
**Pull request by grgkaran03**
# What does this PR do?
Add configuration_vit_mae.py to utils/documentation_tests.txt and configuration_yoso.py for doctest.

Based on issue #19487 

@ydshieh  could you please review it?

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/transformers/models/vit_mae/configuration_vit_mae.py [hunk 1]
```diff
@@ -82,12 +82,12 @@ class ViTMAEConfig(PretrainedConfig):
...
 
     ```python
-    >>> from transformers import ViTMAEModel, ViTMAEConfig
+    >>> from transformers import ViTMAEConfig, ViTMAEModel
 
     >>> # Initializing a ViT MAE vit-mae-base style configuration
     >>> configuration = ViTMAEConfig()
 
-    >>> # Initializing a model from the vit-mae-base style configuration
+    >>> # Initializing a model (with random weights) from the vit-mae-base style configuration
```
Comment (ydshieh) [inline src/transformers/models/vit_mae/configuration_vit_mae.py:90]: Thank you for taking care of removing this previous empty line

**Review by ydshieh — APPROVED** [review]
Thank you @grgkaran03 🚀

**Comment by ydshieh** [comment]
Hi @grgkaran03 Do you intend to work on yoso in this PR? I see you reverted the change in a commit, then added it back in the last commit.

**Comment by grgkaran03** [comment]
Did that by mistake, will rectify the error!

**Comment by grgkaran03** [comment]
Hi! got a little confused. I worked in yoso and vit_mae in this pr, if it's fine...

**Review by ydshieh — APPROVED** [review]
Thanks.

**Pull request merged**